shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java

```diff
@@ -262,6 +276,21 @@ public boolean isAllBindingTables(final Collection<String> logicTableNames) {
         return !result.isEmpty() && result.containsAll(logicTableNames);
     }
     
+    /**
+     * Judge whether logic table is all binding tables.
+     *
+     * @param schema schema
+     * @param sqlStatementContext sqlStatementContext
+     * @param logicTableNames logic table names
+     * @return whether logic table is all binding tables.
```

Comment:
Unnecessary `.` at the end of `return`

Reply:
> 

@menghaoranss Thank you, I will optimize it in next pr.